### PR TITLE
add style colors

### PR DESCRIPTION
**Pull request by TOtterhall**

### CODE REVIEW

File: style/mobile.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+rgba(34,32,28,255) - navigationbar bakgrund
+
+rgba(32,34,32,255) - paragrafer och headlines för telefonerna
+
+
+
+rgba(76,123,195,255) - blåa ramen runt telefonerna i kundvagnsläget
+
+rgba(245,245,245,255) - bakgrunden som displayar skärmarna (den med blåa ramen)
+*/
```
Comment (r-4u-1) [inline style/mobile.css:21]: You could add the colors to css variables

**Review by r-4u-1 — CHANGES_REQUESTED** [review]

**Review by Cappeloow — APPROVED** [review]
Tycker att det ser bra ut

**Review by NavidDev1 — APPROVED** [review]
great

File: style/mobile.css (hunk 1 above)
Comment (TOtterhall) [reply style/mobile.css]: Fixed

**Review by r-4u-1 — APPROVED** [review]

**Pull request merged**